### PR TITLE
Add status badge

### PR DESCRIPTION
**Pull request by hylkealons**
Adds a status badge to the README page

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,15 +1,24 @@
 defmodule ElixirTools.MixProject do
   use Mix.Project
 
+  @version "0.1.0"
```
Comment (dabeat) [inline mix.exs]: What does this have to do with status badge?

Comment (hylkealons) [reply mix.exs]: rebased

**Comment by dabeat** [comment]
:guitar:
